spark/src/main/scala/org/apache/spark/sql/delta/commands/VacuumCommand.scala

```diff
@@ -535,7 +535,7 @@ object VacuumCommand extends VacuumCommandImpl with Serializable {
     // The start and the end commit versions give the range of commit files we want to look into
     // to get the list of eligible files for deletion.
     val eligibleStartCommitVersion = math.min(
-      deltaLog.update().version,
+      snapshot.version,
```

Comment:
Intentional? This isn't a test. It's a bit misleading from the title.

Reply:
Thank you for catching this. This is also a clean up change since this method already takes in a snapshot so there is no need to do `update()` again. I will update the title and description.